source/index.txt

```diff
@@ -163,9 +163,9 @@ Explore MongoDB Realm Services
 Build and Deploy an App
 -----------------------
 
-You can use any of the following interfaces to work with Realm application services:
+You can use any of the following interfaces to work with {+service-short+} application services:
```

Comment:
Here's another one - "App Services application services" is not great - perhaps we could strike any reference to "application services" or "app services" following {+service+} or {+service-short+}?
```suggestion
You can use any of the following interfaces to work with {+service-short+}:
```